src/configs.py

```diff
@@ -196,6 +195,20 @@ def create_fn_params(self):
             "extra_index_urls": extra_index_urls,
         }
 
+    @validator("function_file", pre=True)
+    def validate_function_file(cls, value):
+        if value is None:
+            return value
+
+        try:
+            file_path = Path(value)
```

Comment:
should maybe use `PurePosixPath` given the [SDK docs](https://cognite-sdk-python.readthedocs-hosted.com/en/latest/functions.html#create-function) say the function_path needs to be POSIX format